### PR TITLE
OSOE-807: Removing unneeded MultipleActiveResultSets=True config from the default SQL Server connection string

### LINKED ISSUE #343: Remove MultipleActiveResultSets=True (OSOE-807)
Since it shouldn't really be needed for OC, unless you do parallelized SQL queries within the same connection, what you shouldn't. Related: https://github.com/OrchardCMS/OrchardCore/issues/15224.

[Jira issue](https://lombiq.atlassian.net/browse/OSOE-807)

### PR DESCRIPTION
**Pull request by Piedone**
[OSOE-807](https://lombiq.atlassian.net/browse/OSOE-807)
Fixes #343

[OSOE-807]: https://lombiq.atlassian.net/browse/OSOE-807?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by sarahelsaig — APPROVED** [review]

File: Lombiq.Tests.UI/Docs/Configuration.md [hunk 1]
```diff
@@ -154,7 +154,7 @@ SQL Server on Linux only has SQL Authentication and you still have to tell the t
 ```json
 {
   "SqlServerDatabaseConfiguration": {
-    "ConnectionStringTemplate": "Server=.;Database=LombiqUITestingToolbox_{{id}};User Id=sa;Password=Password1!;MultipleActiveResultSets=True;Connection Timeout=60;ConnectRetryCount=15;ConnectRetryInterval=5;TrustServerCertificate=true;Encrypt=false"
+    "ConnectionStringTemplate": "Server=.;Database=LombiqUITestingToolbox_{{id}};User Id=sa;Password=Password1!;Connection Timeout=60;ConnectRetryCount=15;ConnectRetryInterval=5;TrustServerCertificate=True;Encrypt=false"
```
Comment (sarahelsaig) [inline Lombiq.Tests.UI/Docs/Configuration.md]: I think the true/false here are case insensitive, right? If you've capitalized True for TrustServerCertificate, why not False for Encrypt as well? (same below)

**Review by sarahelsaig — CHANGES_REQUESTED** [review]

File: Lombiq.Tests.UI/Docs/Configuration.md (hunk 1 above)
Comment (Piedone) [reply Lombiq.Tests.UI/Docs/Configuration.md]: Yep, just didn't notice it. Fixed that too.

Comment (sarahelsaig) [reply Lombiq.Tests.UI/Docs/Configuration.md]: It appears you've changed it in SqlServerManager.cs but not in Configuration.md.

```suggestion
    "ConnectionStringTemplate": "Server=.;Database=LombiqUITestingToolbox_{{id}};User Id=sa;Password=Password1!;Connection Timeout=60;ConnectRetryCount=15;ConnectRetryInterval=5;TrustServerCertificate=True;Encrypt=False"
```

Comment (Piedone) [reply Lombiq.Tests.UI/Docs/Configuration.md]: I edited that too, but apparently just typed `false` again, thanks.

**Pull request merged**